src/api/todos.ts

```diff
@@ -5,4 +5,19 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodos = ({ id, title, completed }: Omit<Todo, 'userId'>) => {
+  return client.post<Todo>('/todos', { id, title, completed });
+};
+
+export const updateTodos = ({
```

Comment:
since for this task we have no requirements for updating and deleting I will not write remarks for this methods (yet, pay attention that after page reloading the todo state isn't updated)